ci/pipelines/pull-request/deploy_pr_pipeline.sh

```diff
@@ -68,6 +68,18 @@ for i in ${GEODEBUILDDIR}/test-stubs/*.yml; do
     ${i} > ${TMP_DIR}/${X}
 done
 
+for i in ${SCRIPTDIR}/test-stubs/*.yml; do
```

Comment:
Why not put this in the main pipeline too. It makes it less of a one off in this script and applies the same logic to newly merged tests on the mainline pipeline.